### PR TITLE
STONEBLD-10: set -e to task script properly

### PR DESCRIPTION
**Pull request by tkdchen**
Signed-off-by: Chenxiong Qi <cqi@redhat.com>

### CODE REVIEW

**Comment by Michkov** [comment]
Test tasks are not supposed to fail when there is an issue.

File: task/fbc-related-image-check/0.1/fbc-related-image-check.yaml [hunk 1]
```diff
@@ -31,6 +31,7 @@ spec:
           cpu: 10m
       script: |
         #!/usr/bin/env bash
+        set -e
```
Comment (MartinBasti) [inline task/fbc-related-image-check/0.1/fbc-related-image-check.yaml:34]: We cannot fail script, checks must always return `HACBS_TEST_OUTPUT`

Any error handling must happen within the script

**Review by MartinBasti — CHANGES_REQUESTED** [review]

File: task/fbc-related-image-check/0.1/fbc-related-image-check.yaml (hunk 1 above)
Comment (tkdchen) [reply task/fbc-related-image-check/0.1/fbc-related-image-check.yaml]: Remove.

File: task/configure-build/0.1/configure-build.yaml [hunk 2]
```diff
@@ -33,6 +33,7 @@ spec:
           mountPath: /secret/default-push-secret
       script: |
         #!/usr/bin/env bash
+        set -e
```
Comment (MartinBasti) [inline task/configure-build/0.1/configure-build.yaml:36]: maybe `set -euo pipefail` to have all important failures covered?

Comment (tkdchen) [reply task/configure-build/0.1/configure-build.yaml:36]: Looks unnecessarily for now in this script I think. `-e` is enough for catching potential problems as described in the JIRA issue. Option `pipefail` is added for some other script that has pipe operation, and `-u` is helpful along with the size of script is growing.

Comment (MartinBasti) [reply task/configure-build/0.1/configure-build.yaml:36]: okay

**Review by MartinBasti — APPROVED** [review]

**Pull request closed without merging**